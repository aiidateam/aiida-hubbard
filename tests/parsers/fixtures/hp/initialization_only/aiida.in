&INPUTHP
  determine_num_pert_only = .true.
  iverbosity = 2
  outdir = 'out'
  prefix = 'aiida'
/
