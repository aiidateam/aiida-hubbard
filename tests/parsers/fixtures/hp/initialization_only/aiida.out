
     Program HP v.5.1.1 starts on  6Oct2019 at 16: 0: 0

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org",
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     1 processors

     =--------------------------------------------------------------=

               Calculation of Hubbard parameters from

               density functional perturbation theory

     =--------------------------------------------------------------=

   Info: using nr1, nr2, nr3 values from input

   Info: using nr1s, nr2s, nr3s values from input

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      =  SLA  PW   PBX  PBC ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file O.pbe-n-kjpaw_psl.0.1.UPF: wavefunction(s)  2P renormalized

     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         913     459    149                13597     4819     857



     bravais-lattice index     =             0
     lattice parameter (alat)  =        9.3664 (a.u.)
     unit-cell volume          =      216.7049 (a.u.)^3
     number of atoms/cell      =             4
     number of atomic types    =             3
     kinetic-energy cut-off    =         30.00 (Ry)
     charge density cut-off    =        240.00 (Ry)
     conv. thresh. for NSCF    =       1.0E-11
     conv. thresh. for chi     =       1.0E-05
     Input Hubbard parameters (in eV):
       U ( 1)                     =   3.00000E+00

     celldm(1) =  9.36643  celldm(2) =  0.00000  celldm(3) =  0.00000
     celldm(4) =  0.00000  celldm(5) =  0.00000  celldm(6) =  0.00000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (  0.2839  0.1639  0.9448 )
               a(2) = ( -0.2839  0.1639  0.9448 )
               a(3) = (  0.0000 -0.3278  0.9448 )

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.7614  1.0169  0.3528 )
               b(2) = ( -1.7614  1.0169  0.3528 )
               b(3) = (  0.0000 -2.0339  0.3528 )

     Atoms inside the unit cell (Cartesian axes):
     site n.  atom      mass           positions (alat units)
        1      Co     58.9332   tau( 1) = (  0.00000  0.00000  0.00000  )
        2      Li      6.9410   tau( 2) = ( -0.00000  0.32778  0.47238  )
        3      O      15.9994   tau( 3) = ( -0.00000 -0.00000  0.64281  )
        4      O      15.9994   tau( 4) = (  0.28387  0.16389  0.30194  )

     Atom which will be perturbed:

        1      Co     58.9332   tau( 1) = (  0.00000  0.00000  0.00000  )

     HP           :     1.49s CPU         1.52s WALL


   This run was terminated on:  16: 0: 2   6Oct2019

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
